Logger.md

```diff
@@ -57,22 +56,22 @@ public protocol Logger {
     ///   - level: Type of log.
     ///   - tag: An additional label to help categorise logs.
     ///   - message: The message to be logged.
-    func log(_ level: LogLevel, tag: String?, message: String)
+    func log(_ level: LogLevel, tag: String?, _ message: String)
```

Comment:
My motivation to remove `message` from the method signature was because I thought we could use default parameters in level and tag, being able to do things like `MyLogger.log("Hello world")". As this is not possible, we will be forced always to pass the three parameters in the method. Therefore, maybe is better to show always the three parameter names in the method signature rather than only showing the `tag`one.

Feel free to compose it as you think is best.

Reply:
Done